vite.config.js

```diff
@@ -0,0 +1,32 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import tsconfigPaths from 'vite-tsconfig-paths'
+
+export default defineConfig({
+  base: "",
+  root: "",
+  plugins: [
+    react(),
+    tsconfigPaths()
+  ],
+  build: {
+    sourcemap: true,
+    emptyOutDir: true,
+    rollupOptions: {
+      output: {
+        entryFileNames: "index.js"
```

Comment:
Why do we need this? It is related to  https://github.com/kbss-cvut/fta-fmea-ui/commit/b14257e3022d2cedee28da3c4d1af22b5c5ea901 .